src/test/java/tests/MultiModelTest.java

```diff
@@ -23,7 +23,7 @@ public static void setup() {
         UI.status = new StatusUIStub();
     }
 
-    MultiModel multiModel = new MultiModel(new Preferences(true));
+    MultiModel multiModel = new MultiModel(mock(Preferences.class));
```

Comment:
Why do you use mock for some but test preference for others? Any rationale?

I agree that in unit testing we should use mock for other components, but will Mockito not mess with the application preference directory?